whisper/whisperv6/api.go

```diff
@@ -418,10 +418,14 @@ func (api *PublicWhisperAPI) Messages(ctx context.Context, crit Criteria) (*rpc.
 					}
 				}
 			case <-rpcSub.Err():
-				api.w.Unsubscribe(id)
+				if err := api.w.Unsubscribe(id); err != nil {
+					panic(err)
+				}
 				return
 			case <-notifier.Closed():
-				api.w.Unsubscribe(id)
+				if err := api.w.Unsubscribe(id); err != nil {
+					panic(err)
```

Comment:
It'd be better to fix this one and the one above with an API change to remove the return value. The only error that can happen here is "invalid filter ID", which happens when the filter is already unsubscribed.